### ISSUE TITLE
Unsupported algorithm sha3-256

### ISSUE THREAD

**Issue by 7heMech** [telegraf/telegraf#1987]
I saw another issue was made on this a while ago, but it's already closed.
I use bun runtime and when I try to use telegraf with just `bot.launch()` it doesn't happen, but the issue occurs when I use it with a webhook:
```js
createServer(
  await bot.createWebhook({ domain: "example.com" })
).listen(3000);
```
I get this error
```
168 |     }
      at secretPathComponent (/home/runner/3256c0c4-2976-4267-afcf-79a450257baf/node_modules/telegraf/lib/telegraf.js:170:16)
undefined
170 |         return crypto
      at createHash (node:crypto:1287:81
 (/home/runner/3256c0c4-2976-4267-afcf-79a450257baf/node_modules/telegraf/lib/telegraf.js:132:90)
                     ^
:9:
      at getDomainOpts (/home/runner/3256c0c4-2976-4267-afcf-79a450257baf/node_modules/telegraf/lib/telegraf.js:132:85)
169 |     secretPathComponent() {
      at /home/runner/3256c0c4-2976-4267-afcf-79a450257baf/node_modules/telegraf/lib/telegraf.js:142:33


failed to exec [sh -c bun index.js]: exit status 1
TypeError: Unsupported algorithm sha3-256
      at CryptoHasher (native:1:1)
      at createHash (node:1:1287)
```

**Comment by amirrezasalimi** [comment]
same problem.

**Comment by MKRhere** [comment]
Bun version?

![image](https://github.com/user-attachments/assets/1ac7fb99-118b-45f2-8c19-b5944070e8bc)

**Comment by 7heMech** [comment]
@MKRhere thanks for bringing this to my attention, upon some testing I see that the issue was fixed in `bun-v1.1.5`.
